distribution.md

```diff
@@ -10,23 +10,23 @@ This document specifies the artifact format, delivery mechanism, and order resol
   - [Table of Contents](#table-of-contents)
   - [Distribution API Version](#distribution-api-version)
   - [Artifact Format](#artifact-format)
-    - [Buildpack](#buildpack)
     - [Buildpackage](#buildpackage)
+    - [Lifecycle](#lifecycle)
+    - [Build Image](#build-image)
+    - [Run Image](#run-image)
+    - [Builder Image](#builder-image)
 
 ## Distribution API Version
 
-This document specifies Distribution API version `0.2`.
+This document specifies Distribution API version `0.3`.
 
 Distribution API versions:
  - MUST be in form `<major>.<minor>` or `<major>`, where `<major>` is equivalent to `<major>.0`
  - When `<major>` is greater than `0` increments to `<minor>` SHALL exclusively indicate additive changes
+ - Each Distributable artifact MUST contain the label `io.buildpacks.distribution.api` denoting the distribution API
 
 ## Artifact Format
 
-### Buildpack
-
-A buildpack MUST contain a [`buildpack.toml`](buildpack.md#buildpacktoml-toml) file at its root directory.
-
 ### Buildpackage
 
 A buildpackage MUST exist as either an OCI image on an image registry, an OCI image in a Docker daemon, or a `.cnb` file.
```

Comment:
@sclevine, I wonder if it would be worth removing the `.cnb` file extension declaration and instead simply making the requirement "..., or an uncompressed tar archive in [oci-layout](https://github.com/opencontainers/image-spec/blob/main/image-layout.md) format". 

The file extension seems irrelevant especially since we say it's only a _SHOULD_ meaning it can't be used for an form of validation.